src/js/edu-benefits/1990e/config/form.js

```diff
@@ -34,6 +43,9 @@ const formConfig = {
   transformForSubmit: transform,
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
+  defaultDefinitions: {
+    dateRange: dateRange.schema
```

Comment:
Done...following what we discussed with @annekainicUSDS in https://github.com/department-of-veterans-affairs/vets-website/pull/5056